### ISSUE TITLE
In neural net, the training accuracy in 57% but test accuracy in 93%

### ISSUE THREAD

**Issue by vibhuvardhansingh** [vibhuvardhansingh/Data-Mining-Project#2]

**Comment by vibhuvardhansingh** [comment]
Lowering the training set data otherwise overfitting happens
